kahuna/public/js/usage-rights/usage-rights-editor.html

```diff
@@ -77,6 +76,10 @@
                 <span ng:if="ctrl.saving">Saving</span>
                 <span ng:if="ctrl.saved">Saved</span>
         </button>
+        <button class="ico-button ure__remove" type="button" ng:click="ctrl.remove()"
+            title="remove usage rights overrides">
+            <i class="material-icons">delete</i>
```

Comment:
before merging to master, this'll need to become `<gr-icon>delete</gr-icon>` - `.material-icons` is dead now.